src/js/modules/RoutingApp.js

```diff
@@ -1,29 +1,34 @@
 import Provider from "react-redux/lib/components/Provider";
 import React, { Component } from "react";
 import appHistory from "tools/appHistory";
-import ArticleForm from "./forms/components/ArticleForm";
-import SectionForm from "./forms/components/SectionForm";
+import CreateArticlePage from "./articles/components/CreateArticlePage"
+import SectionForm from "./sections/components/SectionForm";
 import ArticlesTable from "./articles/components/ArticlesTable";
 import UsersList from "./users/components/UsersList";
 import ConnectedRouter from "react-router-redux/ConnectedRouter";
 import store from "../store";
 import MuiThemeProvider from "material-ui/styles/MuiThemeProvider";
-import { Route } from "react-router-dom";
+import { Route, Switch } from "react-router-dom";
 import MainApp from "./main/components/MainApp";
+import NotFoundPage from "./main/components/NotFoundPage"
+import HomePage from "./main/components/HomePage"
 
 class RoutingApp extends Component {
   render() {
     return (
       <MuiThemeProvider>
         <Provider store={store}>
           <ConnectedRouter history={appHistory}>
-            <div>
-              <Route path="/" component={MainApp} />
-              <Route exact path="/articles" component={ArticlesTable} />
-              <Route path="/articles/new" component={ArticleForm} />
-              <Route exact path="/users" component={UsersList} />
-              <Route path="/sections/new" component={SectionForm} />
-            </div>
+            <MainApp>
+              <Switch>
+                <Route exact path="/articles" component={ArticlesTable} />
+                <Route path="/articles/new" component={CreateArticlePage} />
+                <Route exact path="/users" component={UsersList} />
+                <Route path="/sections/new" component={SectionForm} />

```

Comment:
It's a little inconsistent. I would expect `/articles/new` and `/sections/new` to be either named CreateArticlePage and CreateSection or ArticleForm and SectionForm.